src/components/List/NormalList/NormalList.style.ts

```diff
@@ -1,16 +1,20 @@
 import { css, SerializedStyles } from '@emotion/react';
+import { Theme } from 'theme';
 import { rem } from 'theme/utils';
 
 export const listStyle = ({
   width,
   height,
+  isSearchable,
 }: {
   width?: number;
   height?: number;
-}) => (): SerializedStyles => css`
+  isSearchable?: boolean;
+}) => (theme: Theme): SerializedStyles => css`
   padding-left: 0;
   margin-top: 0;
   margin-bottom: 0;
+  border-radius: ${isSearchable ? 'initial' : theme.spacing.xsm};
```

Comment:
why we have special cases for radius ?

Reply:
@panvourtsis Because it causes this to happen 
![image](https://user-images.githubusercontent.com/13350837/143416369-0fae0b35-4fc4-4132-a44c-b7ed43af5c7a.png)